test/sqllogictest/introspection/unstable.slt

```diff
@@ -35,13 +27,7 @@ CREATE VIEW v AS SELECT * FROM mz_internal.mz_scheduling_elapsed
 statement ok
 CREATE TEMPORARY VIEW v AS SELECT * FROM mz_internal.mz_scheduling_elapsed
```

Comment:
```suggestion
CREATE TEMPORARY VIEW v AS SELECT id, worker_id, elapsed_ns FROM mz_internal.mz_scheduling_elapsed
```

Reply:
Thanks! I used `count(*)` instead, to make the test more robust against changes to the columns.